### DISCUSSION TITLE
New plugins

### DISCUSSION THREAD

**Discussion by 1stcall** [gitbls/sdm#147] [Ideas]
Clone a git repo would be cool.  ie. --plugin gitclone "repo=https://github.com/xxxx/yyy|dest=/home/myuser/mygits/yyy"

Add sources to apt. ie. --plugin addsources "repo=https://anyrepo.net/reponame|gpgkey=https//anyrepo.net/reponame/thekey.gpg or path to local key"

**Comment by gitbls** [comment]
Both excellent ideas. Thx!

**Comment by 1stcall** [comment]
I did consider '--plugin instscp "https://raw.githubusercontent.com/gitbls/sdm/master/EZsdmInstaller"` but the security implications would be dangerous.  Would have to come with CAPITOL LETTER warning.  The repo one is bad enough, but at least we would have gpg protection.
